gslib/tests/test_cp.py

```diff
@@ -1382,6 +1371,20 @@ def test_cross_provider_large_cp(self):
       # Ensure copy also works across json upload chunk boundaries.
       self.RunGsUtil(['cp', suri(s3_key), suri(gs_bucket)])
 
+  @unittest.skipUnless(HAS_S3_CREDS, 'Test requires both S3 and GS credentials')
+  def test_gs_to_s3_multipart_cp(self):
+    """Ensure daisy_chain works for an object that is downloaded in 2 parts."""
+    s3_bucket = self.CreateBucket(provider='s3')
+    gs_bucket = self.CreateBucket(provider='gs', prefer_json_api=True)
+    gs_key = self.CreateObject(bucket_uri=gs_bucket,
+                               contents=b'b' * 101 * 1024 * 1024,
```

Comment:
Can we maybe define this in relation to `_DEFAULT_DOWNLOAD_CHUNK_SIZE` in `gslib/daisy_chain_wrapper.py`?  E.g. `num_bytes = int(_DEFAULT_DOWNLOAD_CHUNK_SIZE * 1.1)`.  Without doing so, if we ever change that constant, it would be easy to forget to change this test too.

Reply:
Done